src/modules/tests/test-data/cat-d/pcv-door-exercise/pcv-door-exercise.reducer.ts

```diff
@@ -0,0 +1,61 @@
+import { CatDUniqueTypes } from '@dvsa/mes-test-schema/categories/D';
+import * as pcvDoorExerciseActions from './pcv-door-exercise.actions';
+import { createFeatureSelector } from '@ngrx/store';
+
+export const initialState: CatDUniqueTypes.PcvDoorExercise = {
```

Comment:
For correctness instead of using CatDUniqueTypes.PcvDoorExercise we should have a type as follows and use that. Functionally it doesn't make a difference as it's all the same behind the scenes

`export type PCVDoorExercise = 
| CatDUniqueTypes.PcvDoorExercise
| CatD1UniqueTypes.PcvDoorExercise
| CatDEUniqueTypes.PcvDoorExercise
| CatD1EUniqueTypes.PcvDoorExercise`